bundler.d/development.rb

```diff
@@ -15,4 +15,11 @@
   gem 'spring', '>= 1.0', '< 3'
   gem 'benchmark-ips'
   gem 'foreman'
+  if ENV['RAILS_ENV'] == 'development' && RUBY_VERSION >= '2.3'
```

Comment:
this will only work if you explicitly set RAILS_ENV (which I think most devs don't do as its the default).

Reply:
I guess you can't use `Rails.env` yet?

Reply:
Yeah Rails is not initialized yet. I thought that RAILS_ENV is always set, looks like it is not.